packages/templates/blog/template/src/api/about/content-types/about/schema.json

```diff
@@ -0,0 +1,27 @@
+{
+  "kind": "singleType",
+  "collectionName": "abouts",
+  "info": {
+    "singularName": "about",
+    "pluralName": "abouts",
+    "displayName": "about"
```

Comment:
Could you please capitalize the display name, and add a description? (same comment for all content types)